src/common/views/opds.ts

```diff
@@ -52,6 +52,8 @@ export interface OpdsResultView {
     navigation?: OpdsLinkView[];
     publications?: OpdsPublicationView[];
     searchUrl?: string;
+    nextPageUrl?: string;
+    previousPageUrl?: string;
```

Comment:
TODO: implement support for `first` and `last` link relations as well?

Reply:
Do you have opds feed with first and last link ?

Reply:
No.
But the `first` and `last` link relations are in the OPDS/Atom specifications:
https://tools.ietf.org/html/rfc5005#section-3
https://drafts.opds.io/opds-2.0#3-pagination
https://tools.ietf.org/html/rfc5988#section-6.2.2

Reply:
The Feedbooks OPDS feed has `previous` / `next` pagination, but not `first` / `last`, as far as I can tell:
http://readium2.herokuapp.com/opds-v1-v2-convert/https%3A%2F%2Fcatalog.feedbooks.com%2Fpublicdomain%2Fbrowse%2Fen%2Ftop.atom%3Fcat%3DFBFIC009000%26page%3D2